test/JDBC/expected/datareader_datawriter.out

```diff
@@ -239,19 +239,19 @@ go
 alter role db_datawriter add member db_roles_r1; 
 go
 
-select count(*) from sys.database_principals where name like '%db_datareader%';
+select name, is_fixed_role from sys.database_principals where name like '%db_datareader%' order by name;
 go
```

Comment:
1. Were there any other system objects which shows is_fixed_role metadata?
2. There is a possibility of refactoring of other system objects using this is_fixed_role column. We might get some perf. boost there. Let's track it separately.

Reply:
1. Only server_principals and database_principals shows `is_fixed_role` metadata.
2. Will create a JIRA to track it.